webpack.config.js

```diff
@@ -25,8 +25,13 @@ module.exports = {
     publicPath: '/',
     filename:   'bundle.js'
   },
+  devtool: 'inline-source-map',
   devServer: {
-    contentBase: './dist'
+    hot: true,
```

Comment:
Eh actually not quite sure.. got from here: https://github.com/bananaoomarang/isomorphic-redux/blob/master/webpack.config.js

The inline-source-map option allows you to see which source file errors come from though, instead of just bundle.js